frontend/cli_to_rails_proxy.js

```diff
@@ -1,9 +1,10 @@
 const PROXY_HOSTNAME = process.env.PROXY_HOSTNAME || 'localhost';
```

Comment:
@machisuji So it should be:
```suggestion
const PROXY_HOSTNAME = process.env.PROXY_HOSTNAME || process.env.HOST || 'localhost';
```